js/script.js

```diff
@@ -444,7 +468,7 @@ function setPreviewFollow(mouseLoc){
 }
 
 function itemClick() {
-    createPreview(this.dataset.url, this.dataset.r, this.dataset.direction, this.dataset.w, this.dataset.h);
+    createPreview(this.dataset.url, this.dataset.r, this.dataset.direction, this.dataset.w, this.dataset.h, this.dataset.mirrorFlippedHorizontal);

```

Comment:
Seems like this method uses a lot of values from dataset. It might be easier to pass the dataset object it's self